klog_file.go

```diff
@@ -116,6 +117,18 @@ func logName(tag string, t time.Time) (name, link string) {
 	return name, program + "." + tag
 }
 
+func rotateLogName(logFile string, index uint64, t time.Time) string {
+	return fmt.Sprintf("%s.%04d%02d%02d-%02d%02d%02d.%d",
```

Comment:
How about to add new flag set rotate log file time format?  @dims

Reply:
@yuzhiquan we can do that in a follow up?

Reply:
> @yuzhiquan we can do that in a follow up?

Sure, it's useful for someone,like me,to set specified time format for rotate log file, but i do not known it's helpful or not?